src/test/java/com/hyperlocal/server/Record.java

```diff
@@ -0,0 +1,277 @@
+package com.hyperlocal.server;
+
+import com.github.jasync.sql.db.RowData;
+
+import org.joda.time.LocalDateTime;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+public class Record implements RowData {
```

Comment:
Would it be better to name this FakeRowData since its not a real implementation of RowData